src/shared/approval-status/get-approval-status.js

```diff
@@ -37,7 +38,21 @@ const getApprovalStatusIcon = approvalStatus => {
     }
 }
 
-const getApprovalStatusText = approvalStatus => {
+const getApprovedStatusText = ({ approvedAt, approvedBy }) => {
+    const approvedMsg = approvedBy
+        ? i18n.t('Approved by {{approvedBy}}', { approvedBy })
```

Comment:
If `approvedBy` contains special characters (e.g. `Name O'Lastname`), these will be HTML escaped. So we need to unescape like so:

```suggestion
        ? i18n.t('Approved by {{- approvedBy}}', { approvedBy })
```